homeassistant/components/zerproc/strings.json

```
@@ -0,0 +1,14 @@
+{
+  "title": "Zerproc",
+  "config": {
+    "step": {
+      "confirm": {
+        "description": "Do you want to set up Zerproc lights?"
+      }
+    },
+    "abort": {
+      "single_instance_allowed": "Only a single configuration of Zerproc lights is necessary.",
+      "no_devices_found": "No Zerproc lights found nearby."
```

I've just merged an update to our scaffold script that lets this use references.
```suggestion
        "description": "[%key:common::config_flow::description::confirm_setup%]"
      }
    },
    "abort": {
      "single_instance_allowed": "[%key:common::config_flow::abort::single_instance_allowed%]",
      "no_devices_found": "[%key:common::config_flow::abort::no_devices_found%]"
```